2-mandatory/4-stocks.js

```diff
@@ -64,10 +87,22 @@ function getPriceChanges(closingPricesForAllStocks) {
     The price should be shown with exactly 2 decimal places.
 */
 function highestPriceDescriptions(closingPricesForAllStocks, stocks) {
-    // TODO
+    let myArr = closingPricesForAllStocks;
```

Comment:
This is a nice implementation. There is an alternative you might be interested in - which uses Javascript's Math.max method. You can see this in the Week 3 Solutions repo. Also you can read more here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max